### DISCUSSION TITLE
I have a MetaMask connecting issue only when it's first render on PC Chrome, even rainbowkit.com

### DISCUSSION THREAD

**Discussion by DongwooSohn** [rainbow-me/rainbowkit#1390] [Feedback]
I'll explain my case of the issue.

1. Open a new chrome tab.
2. Go to "rainbowkit.com"
3. click "Connect Wallet" button at top right.
4. click MetaMask

After number 4, it's supposed to be connected but nothing happened. (just MetaMask button turn into blue)
![스크린샷 2023-07-10 오후 2 29 19](https://github.com/rainbow-me/rainbowkit/assets/110084028/83963ad1-1365-4dde-9542-2aeffba558b0)

It occurs only when it's first render.
After refreshing the page, it works fine.

As the same situation, I can't sign message at the first render. (in my test web)
It also works fine after refreshing the page.
And console says "ConnectorNotFoundError: Connector not found".

Does anybody have a idea about this issue?

**Comment by DanielSinclair** [comment]
This is an issue with Wagmi, and likely the result of a small mistake in your Wagmi configuration/usage. If you're still seeing the problem, please open an issue on the Wagmi or RainbowKit projects with a reproducible example.
